Support/cmp/Modules/FindTBB.cmake

```diff
@@ -153,14 +153,14 @@ if(NOT _TBB_INSTALL_DIR)
     endif(_TBB_DEFAULT_INSTALL_DIR)
 endif(NOT _TBB_INSTALL_DIR)
 # sanity check
-if(NOT _TBB_INSTALL_DIR)
+if(NOT _TBB_INSTALL_DIR AND WIN32)
```

Comment:
It breaks configuring when TBB_INSTALL_DIR is not set, which is not needed when TBB could be found at the Linux default path (/usr).